src/absil/il.fsi

```diff
@@ -1271,7 +1273,6 @@ and [<NoComparison; NoEquality>]
     member MethodImpls: ILMethodImplDefs
     member Events: ILEventDefs
     member Properties: ILPropertyDefs
-    member CustomAttrs: ILAttributes
```

Comment:
If you're going to change the name from `CustomAttrs` to `CustomAttributes` then do it everywhere for every type in `il.fsi`/`il.fs` for consistency.  But suggest you do that in a separate PR.

Reply:
I'm going to leave it as `CustomAttrs`. Not sure why I wanted to change it to `CustomAttributes`. You are right I would need to change it everywhere if I were to rename it. We can do that later if we decide to do it, but not for this PR as I've decided.